script.js

```diff
@@ -0,0 +1,19 @@
+const toggle = document.querySelector('#menu-btn');
+const menu = document.querySelector('ul');
+
+toggle.onclick = () => {
+  if (menu.classList.contains('active')) {
+    toggle.classList.remove('fa-times');
+    menu.classList.remove('active');
+    toggle.style.color = '#6070ff';
+  } else {
+    toggle.classList.add('fa-times');
+    menu.classList.add('active');
+    toggle.style.color = '#fff';
+  }
+};
```

Comment:
Everything is working perfectly 💯, However, when the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed 👍. `but when I scroll back to the top of the page I cannot see the mobile menu anymore` 👀😳. 

I found out that the reason for that was because you changed its color to white. `#ffffff` with JS. 🤓

Your design
![image](https://user-images.githubusercontent.com/78251927/230968861-90242206-472b-42fa-9aab-f05fab214573.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/230969698-3398dad8-40c9-46e1-90b5-0d718e12e219.png)

Please kindly adjust the color so that your design matches [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins) and everything works perfectly. 💪